build/ci/release.yml

```diff
@@ -70,6 +70,56 @@ functions:
       params:
         <<: *go_options
         binary: build/package/generate-notices.sh
+  "generate sbom":  
+    - command: ec2.assume_role
+      params:
+        role_arn: ${ecr_role_arn}
+    - command: subprocess.exec
+      params:
+        <<: *go_options
+        include_expansions_in_env:
+          - workdir
+        binary: make
+        args:
+          - gen-purls
```

Comment:
I think we shouldn't do it anymore, we should make sure the file is fully updated